stage1/init/common/units.go

```diff
@@ -50,11 +50,27 @@ func MutableEnv(p *stage1commontypes.Pod) error {
 		unit.NewUnitOption("Unit", "DefaultDependencies", "false"),
 		unit.NewUnitOption("Unit", "Requires", "systemd-journald.service"),
 		unit.NewUnitOption("Unit", "After", "systemd-journald.service"),
+		unit.NewUnitOption("Unit", "Wants", "pod-ready.service"),
 		unit.NewUnitOption("Unit", "Before", "halt.target"),
 		unit.NewUnitOption("Unit", "Conflicts", "halt.target"),
 	)
 
 	w.WriteUnit(
+		ServiceUnitPath(p.Root, "pod-ready"),
+		"failed to write pod-ready",
+		unit.NewUnitOption("Unit", "Description", "rkt pod-ready signaling"),
+		unit.NewUnitOption("Unit", "DefaultDependencies", "false"),
+		unit.NewUnitOption("Unit", "Requires", "default.target"),
+		unit.NewUnitOption("Unit", "After", "default.target"),
+		unit.NewUnitOption("Unit", "Before", "halt.target"),
+		unit.NewUnitOption("Unit", "Conflicts", "halt.target"),
+		unit.NewUnitOption("Service", "Type", "oneshot"),
+		unit.NewUnitOption("Service", "Restart", "no"),
+		unit.NewUnitOption("Service", "ExecStart", `/bin/bash -c "systemd-tmpfiles --create  <(echo L+ /rkt/supervisor-status - - - - ready)"`),
+		unit.NewUnitOption("Service", "ExecStop", `/bin/bash -c "systemd-tmpfiles --remove  <(echo R /rkt/supervisor-status)"`),
```

Comment:
As discussed OOB, it makes sense not to generate this unit file programmatically, but add it statically in the image since this is actually needed for both the immutable and mutable pod environment and doesn't need any dynamic configuration.